README.md

```diff
@@ -55,3 +55,76 @@ You can install them using the following command:
 sudo apt update
 sudo apt install -y libssl-dev pkg-config
 ```
+
+## Step-by-Step Guides
+
+### Prerequisites
+Ensure you have the following installed:
+ * Rust
+ * Cargo
+
+## Setup
+1. Create a working directory eg(cd didcomm-mediator-rs) and cd into your directory.
+
+ 2. Clone the repository using the following command:
+
+git clone https://github.com/adorsys/didcomm-mediator-rs.git
+
+## Configuration Options
+The following configuration options are available:
+* `--config-file`: Path to the configuration file. Defaults to `config.toml`.
+* `--config`: Configuration options in TOML format. Overrides the configuration file.
+* `--log-level`: Log level. Can be one of `trace`, `debug`, `
+info`, `warn`, `error`, or `off`. Defaults to `info`.
```

Comment:
I dont't think this is neccessary

Reply:
@Christiantyemele Should it be removed or what?

Reply:
Yes i think it should be removed